core/unix/os.c

```diff
@@ -3182,7 +3178,7 @@ replace_thread_id(thread_id_t old, thread_id_t new)
     thread_id_t new_tid = new;
     ASSERT(is_thread_tls_initialized());
     DOCHECK(1, {
-        thread_id_t old_tid;
+        thread_id_t old_tid = 0;
```

Comment:
Good to not leave these uninitialized, but just curious whether it caused any visible issue? Since these are anyway written in the READ macro.

Reply:
I remember it causes compile errors if leaves uninitialized, I'll recheck later.

Reply:
Only part of them are necessary, but I think we can initialize them anyway.